docs/source/models/field_ionization.rst

```diff
@@ -56,6 +56,24 @@ Overview: Implemented Models
 
     Models marked with "(R&D)" are under *research and development* and should be used with care.
 
+Ionization Current
+------------------
+
+In order to conserve energy, PIConGPU supports an ionization current to weaken the electric field for every field ionization model.
```

Comment:
*to weaken* --> *[...] to decrease the electric field according to the amount of energy lost to field ionization processes*